js/Board.js

```diff
@@ -0,0 +1,115 @@
+class Board {
+  constructor(slots, playerBtns) {
+    this.slots = slots;
+    this.playerBtns = playerBtns;
+
+    this.player1 = new Player('bg-danger');
```

Comment:
I think `bg-danger` and `bg-warning` are bootstrap-isms that in this context are sort of confusing. Would probably help to rename them `yellow` and `red` or something more domain-specific for this game.